app/views/layouts/header.pug

```diff
@@ -31,12 +31,11 @@ body
         a(href='/')
           img(src='/img/bird.svg')
       if (req.isAuthenticated())
-        div.new-tweet-container
-          a.new-tweet.btn(data-toggle='modal', href='#myModal')
-            i(class="fa fa-paper-plane")
+        div.new-tweet
+          a.new-tweet__button.btn(data-toggle='modal', href='.tweet-modal')
```

Comment:
Usually, the convention is to use id as target for modal which makes sense because ids are unique and classes are general and repeated.

Reply:
That's fair